sofort/model.py

```diff
@@ -116,7 +116,26 @@ class TransactionDetailsModel(Model):
     costs = ModelType(CostsModel)
     su = ModelType(SuModel)
     status_history_items = SofortListType('status_history_item',
-                                          ModelType(StatusHistoryItemModel))
+                                          ModelType(StatusHistoryItemModel)))
```

Comment:
Travis CI said that you have `SyntaxError` here

Reply:
Fixed, thanks!